docs/infrastructure/Oracles.md

```diff
@@ -0,0 +1,138 @@
+---
+sidebar_position: 30
+sidebar_label: Oracles
+title: Oracles
+---
+
+# Ecosystem Price Oracles, powered by DIA
+
+
```

Comment:
In Markdown, 2 (or more) empty lines are the same as 1. This document includes many unnecessary extra empty lines which might cause issues with translators. I'd recommend removing them, although it's not a biggy

Reply:
fixed